### PR TITLE
Vitaly/feature/admin 

### PR DESCRIPTION
**Pull request by Ziles131**

### CODE REVIEW

File: app/controllers/products.py [hunk 1]
```diff
@@ -176,9 +176,12 @@ def in_selected_category(category_path):
                     collection_id = collection_names[collection_name]
 
                     log(log.DEBUG, "price: %s", product.price)
+                    title = product.title
+                    if Configuration.get_value(shop_id, "LEAVE_VIDAXL_PREFIX"):
```
Comment (tsyboom) [inline app/controllers/products.py]: define as separate variable (const) before for-loop

File: app/controllers/products.py [hunk 2]
```diff
@@ -176,9 +176,12 @@ def in_selected_category(category_path):
...
+                        title = title.replace("vidaXL ", "") if title.startswith("vidaXL ") else title
```
Comment (tsyboom) [inline app/controllers/products.py]: case sensitive!

File: app/views/shopify_bp.py [hunk 3]
```diff
@@ -31,8 +30,11 @@ def install():
     session = shopify.Session(shop_url, version=current_app.config["VERSION_API"])
     log(log.DEBUG, "session: [%s]", session)
     scope = ["write_products", "read_products", "read_script_tags", "write_script_tags"]
+    # permission_url = session.create_permission_url(
+    #     scope, url_for("shopify_bp.finalize", _external=True)
+    # )
```
Comment (tsyboom) [inline app/views/shopify_bp.py:34]: please remove old code!

File: app/views/shopify_bp.py [hunk 4]
```diff
@@ -54,9 +56,10 @@ def finalize():
     log(log.DEBUG, "session: [%s]", shopify_session)
     token = shopify_session.request_token(request.args)
     log(log.DEBUG, "token: [%s]", token)
-    shop = Shop(name=shop_url, access_token=token)
-    db.session.add(shop)
-    db.session.commit()
+    shop = Shop.query.filter(Shop.name == shop_url).first()
+    if not shop:
+        shop = Shop(name=shop_url, access_token=token)
+        shop.save()
```
Comment (tsyboom) [inline app/views/shopify_bp.py:61]:
```python
else:
    shop.access_token = token

File: tests/test_vida_xl.py [hunk 5]
```diff
@@ -37,6 +37,17 @@ def test_get_products(client):
...
+        ):
+            name = title
+        else:
+            if title.startswith("vidaXL "):
+                name = title.replace("vidaXL ", "")
+            else:
+                name = title
+        assert name
         if i > 1111:
             break
```
Comment (tsyboom) [inline tests/test_vida_xl.py:52]: What is a properties of this test?

**Review by kipod — APPROVED** [review]

**Pull request merged**